avro-converter/src/test/java/io/confluent/connect/avro/AvroDataTest.java

```diff
@@ -835,6 +839,27 @@ public void testToConnectRecordWithMetadata() {
   }
 
   @Test
+  public void testToConnectSchemaOptional(){
+    org.apache.avro.Schema avroSchema = org.apache.avro.SchemaBuilder
+               .record("test_record")
+               .fields()
+               .name("field_name").type().nullable().stringType().noDefault()
+               .endRecord();
+    org.apache.avro.generic.GenericData.Record avroRecord =
+       new org.apache.avro.generic.GenericRecordBuilder(avroSchema)
+                .set("field_name", null)
+                 .build();
+
+    Schema connectSchema = SchemaBuilder.struct()
```

Comment:
above you have org.apache.avro.Schema

here you have just Schema

Make them consistent.